src/sysctl/sysctl.c

```diff
@@ -28,7 +28,11 @@
 #include "sysctl-util.h"
 
 static char **arg_prefixes = NULL;
-static bool arg_cat_config = false;
+static enum {
+        CAT_CONFIG_NO,
+        CAT_CONFIG,
+        CAT_CONFIG_TLDR,
+} arg_cat_config = CAT_CONFIG_NO;
```

Comment:
and another time the same enum that is like the regular flags enum. let's just do the regular flags enum please, and reuse it everywhere. A ton easier, and fully systematic